src/libraries/System.Runtime/tests/System/DoubleTests.cs

```diff
@@ -731,6 +731,9 @@ public static IEnumerable<object[]> ToString_TestData()
         {
             yield return new object[] { -4567.0, "G", null, "-4567" };
             yield return new object[] { -4567.89101, "G", null, "-4567.89101" };
+            yield return new object[] { -0.001, "+0.00;-0.00", null, "+0.00" };
```

Comment:
I think this needs to print `-0.00` given that `-0.001` rounds to `-0` and not `+0`

Reply:
Fixed by [dab3fe800c27d81f217e09288824fbb9f2afc274](https://github.com/dotnet/runtime/pull/85565/commits/dab3fe800c27d81f217e09288824fbb9f2afc274) commit

Reply:
The section was written without the consideration around `-0.0` and its relevance to floating-point code. We only made the push towards ensuring things were IEEE 754 compliant starting around .NET Core 2.1

The general intent of `-0.0` is to represent a very small negative number that was rounded towards zero, but which may not itself be zero. Preserving this semantic and treating it as negative is generally desirable and consistent with the handling we expose elsewhere.

We should likely update the docs to cover that:
* One section - The format string applies to all values.
* Two sections - The first section applies to positive values and zeros, and the second section applies to negative values including negative zero for IEEE 754 floating-point types
* Three sections - The first section applies to positive values, the second section applies to negative values, and the third section applies to zero including negative zero for IEEE 754 floating-point types

That still preserves the ability to differentiate between `+0` vs `-0` or to differentiate them based on sign as appropriate, with the default preserving the same behavior as occurs for regular format strings.

CC. @jeffhandley for secondary input -- This may be something we want to take to API review for secondary weigh in; but the original intent when we made the IEEE 754 compat fixes back in 3.0 was along the lines of the above.

Reply:
I support the approach @tannergooding described there, which as I understand it, keeps our current functional behavior intact and adjusts the docs to reflect the behavior we've had for several years now.

I don't think we need to float this past API review unless we're considering a change in functional behavior or a new concept.